packages/beacon-node/src/network/core/networkCoreWorkerHandler.ts

```diff
@@ -81,6 +84,10 @@ export class WorkerNetworkCore implements INetworkCore {
       reqRespBridgeEventDirection
     );
 
+    Thread.errors(modules.networkApiThread).subscribe((err) => {
+      this.modules.logger.error("Network worker thread error", undefined, err);
```

Comment:
I think we usually pass an empty object if there is no context, result would be the same but for consistency reasons might wanna update this

```suggestion
      this.modules.logger.error("Network worker thread error", {}, err);
```